Tests/Marketplace/approved_categories.json

```diff
@@ -15,6 +15,7 @@
 	"Messaging",
 	"Network Security",
 	"Utilities",
-	"Vulnerability Management"
+	"Vulnerability Management",
+	"Email"
```

Comment:
You've already added that to the approved categories no?
Please merge from master and verify